plugins/woocommerce/includes/wc-template-hooks.php

```diff
@@ -255,7 +255,7 @@
  * @see woocommerce_demo_store()
  */
 add_action( 'wp_footer', 'wc_print_js', 25 );
-add_action( 'wp_footer', 'woocommerce_demo_store' );
+add_action( 'wp_body_open', 'woocommerce_demo_store' );
```

Comment:
This change moves the store notice to the top of the page, making it the first focusable element.